ui/modal/info_modal.go

```diff
@@ -147,6 +147,9 @@ func (in *InfoModal) SetupWithTemplate(template string) *InfoModal {
 		customTemplate = setupMixerInfo(in.Theme)
 	case WalletBackupInfoTemplate:
 		customTemplate = backupInfo(in.Theme)
+	case AllowUnmixedSpendingTemplate:
+		title = "Confirm to allow spending from unmixed accounts"
+		customTemplate = allowUnspendUnmixedAcct(in.Theme)
```

Comment:
This is redundant
Since the layout of the text in the text input modal is handled by a function, it does not need an info modal template